flakes.yaml

```diff
@@ -11,3 +11,6 @@ test/new-e2e/tests/containers:
   - TestECSSuite/TestCPU/metric___container.cpu.usage{^ecs_container_name:stress-ng$}
   - TestEKSSuite/TestCPU/metric___container.cpu.usage{^kube_deployment:stress-ng$,^kube_namespace:workload-cpustress$}
   - TestKindSuite/TestCPU/metric___container.cpu.usage{^kube_deployment:stress-ng$,^kube_namespace:workload-cpustress$}
+
+test/new-e2e/tests/installer:
+  - TestPackages/upgrade_scenario_ubuntu_22_04_x86_64/TestUpgradeSuccessful
```

Comment:
I'm surprised this got included in an automated bump PR, just double checking if this was intended!

Reply:
I manually added it, easier this way as it is only one family of os on a test used cross platforms

Reply:
I see the PR description and title are misleading. I used the automated PR as it contained the incident fix. Updating both title and description